runelite-api/src/main/java/net/runelite/api/Varbits.java

```diff
@@ -818,12 +818,28 @@ public final class Varbits
 
 	public static final int SLAYER_POINTS = 4068;
 	public static final int SLAYER_TASK_STREAK = 4069;
+	public static final int SLAYER_WILDERNESS_TASK_STREAK = 5617;
 
 	/**
 	 * The assigned boss for boss slayer.
 	 */
 	public static final int SLAYER_TASK_BOSS = 4723;
 
+	/**
+	 * The slayer master that assigned the task. Varbit does not get reset to 0 after task completion.
+	 * <p>
+	 * 0 = no slayer master
```

Comment:
Would you mind updating this list such that lines have breaks inserted (eg. add `<br>` at the end of these lines so the javadoc won't just render newlines as spaces) and with post-wgs slayer master names